integration/simulation/network/obscuro_node_utils.go

```diff
@@ -148,35 +161,33 @@ func startStandaloneObscuroNodes(params *params.SimParams, stats *stats.Stats, g
 	}
 
 	// round-robin the wallets onto the different obscuro nodes, register them each a viewing key
-	walletClients := make(map[string]rpcclientlib.Client)
-	var i int
+	walletClients := make(map[string][]rpcclientlib.Client)
 	for _, w := range params.Wallets.SimObsWallets {
-		client, err := rpcclientlib.NewViewingKeyNetworkClient(nodeRPCAddresses[i%len(nodeRPCAddresses)])
-		if err != nil {
-			panic(err)
-		}
-		walletClients[w.Address().String()] = client
-		err = viewkey.GenerateAndRegisterViewingKey(client, w)
-		if err != nil {
-			panic(err)
-		}
-		i++
+		walletClients[w.Address().String()] = createAuthenticatedClientsForWallet(nodeRPCAddresses, w)
 	}
 	for _, t := range params.Wallets.Tokens {
 		w := t.L2Owner
-		client, err := rpcclientlib.NewViewingKeyNetworkClient(nodeRPCAddresses[i%len(nodeRPCAddresses)])
+		walletClients[w.Address().String()] = createAuthenticatedClientsForWallet(nodeRPCAddresses, w)
+	}
+
+	return obscuroClients, walletClients
+}
+
+// createAuthenticatedClientsForWallet takes a wallet and sets up a client for it for every node
+func createAuthenticatedClientsForWallet(nodeRPCAddresses []string, wal wallet.Wallet) []rpcclientlib.Client {
```

Comment:
@BedrockSquirrel I found the naming of this confusing. It's really just `createInMemoryClientsForWallet`, but for standalone nodes, right? But the naming makes them sound different.

Can you place the two methods next to each other in the file, and give them more similar nodes - e.g. `createInMemoryClientsForWallet` and `createStandaloneClientsForWallet` (or `createRPCClientsForWallet`).